Index: cobol/orcabt/ORCR0030.CBL
===================================================================
RCS file: /var/lib/cvs/jma-receipt/cobol/orcabt/ORCR0030.CBL,v
retrieving revision 1.10
diff -u -r1.10 ORCR0030.CBL
--- cobol/orcabt/ORCR0030.CBL	6 Nov 2002 06:49:53 -0000	1.10
+++ cobol/orcabt/ORCR0030.CBL	11 Nov 2002 13:32:31 -0000
@@ -629,6 +629,11 @@
                PERFORM     210-HENSYU-SEC
       *
                PERFORM     RECE02-READ-SEC
+	       IF  CNT-IN              =       300
+	           MOVE  ZERO          TO      CNT-IN
+                   PERFORM 900-DBCOMMIT-SEC
+                   PERFORM 900-DBSTART-SEC
+               END-IF
            END-PERFORM
       *
            CLOSE   RECE02-FILE
@@ -3696,4 +3701,33 @@
        RECE02-READ-EXT.
            EXIT.
       *
-      
\ No newline at end of file
+      
+      *****************************************************************
+      *    ＤＢ　ＣＯＭＭＩＴ処理
+      *****************************************************************
+       900-DBCOMMIT-SEC            SECTION.
+      *
+           MOVE    "DBCOMMIT"          TO  MCP-FUNC.
+           CALL    "MCPSUB"          USING
+                                        MCPAREA
+                                        MCPDATA-REC.
+      *
+           .
+       900-DBCOMMIT-EXT.
+           EXIT.
+      *      
+      *****************************************************************
+      *    ＤＢ　ＳＴＡＲＴ処理
+      *****************************************************************
+       900-DBSTART-SEC                SECTION.
+      *
+           MOVE    "DBSTART"           TO  MCP-FUNC.
+           CALL    "ORCMCPSUB"          USING
+                                        MCPAREA
+                                        ORCMCPAREA
+                                        MCPDATA-REC.
+           .
+       900-DBSTART-EXT.
+           EXIT.
+      *      
+      *****************************************************************
Index: cobol/orcabt/ORCR0100.CBL
===================================================================
RCS file: /var/lib/cvs/jma-receipt/cobol/orcabt/ORCR0100.CBL,v
retrieving revision 1.12
diff -u -r1.12 ORCR0100.CBL
--- cobol/orcabt/ORCR0100.CBL	8 Nov 2002 08:37:28 -0000	1.12
+++ cobol/orcabt/ORCR0100.CBL	11 Nov 2002 13:32:31 -0000
@@ -861,6 +861,11 @@
                        PERFORM 2001-SYORI-SEC
       *
                        PERFORM 990-RECE02-READ-SEC
+	       IF  CNT-RECE02          =       300
+	           MOVE  ZERO          TO      CNT-RECE02
+                   PERFORM 900-DBCOMMIT-SEC
+                   PERFORM 900-DBSTART-SEC
+               END-IF
                    END-PERFORM
                END-PERFORM    
            END-PERFORM
@@ -3718,3 +3723,31 @@
            .
        900-DBCLOSE-EXT.
            EXIT.
+      *****************************************************************
+      *    ＤＢ　ＣＯＭＭＩＴ処理
+      *****************************************************************
+       900-DBCOMMIT-SEC            SECTION.
+      *
+           MOVE    "DBCOMMIT"          TO  MCP-FUNC.
+           CALL    "MCPSUB"          USING
+                                        MCPAREA
+                                        MCPDATA-REC.
+      *
+           .
+       900-DBCOMMIT-EXT.
+           EXIT.
+      *      
+      *****************************************************************
+      *    ＤＢ　ＳＴＡＲＴ処理
+      *****************************************************************
+       900-DBSTART-SEC                SECTION.
+      *
+           MOVE    "DBSTART"           TO  MCP-FUNC.
+           CALL    "ORCMCPSUB"          USING
+                                        MCPAREA
+                                        ORCMCPAREA
+                                        MCPDATA-REC.
+           .
+       900-DBSTART-EXT.
+           EXIT.
+      *      
Index: etc/dbgroup.inc
===================================================================
RCS file: /var/lib/cvs/jma-receipt/etc/dbgroup.inc,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 dbgroup.inc
--- etc/dbgroup.inc	4 Sep 2002 07:01:43 -0000	1.1.1.1
+++ etc/dbgroup.inc	11 Nov 2002 13:32:31 -0000
@@ -10,8 +10,8 @@
  priority 100;
  type "PostgreSQL";
  port "localhost";
- name "panda";
- file "orca.log";
+ name "orca";
+ file "/var/lib/jma-receipt/dbredirector/orca.log";
 };
 db_group "shell" {
  type "shell";
Index: etc/jma-receipt.env.in
===================================================================
RCS file: /var/lib/cvs/jma-receipt/etc/jma-receipt.env.in,v
retrieving revision 1.8
diff -u -r1.8 jma-receipt.env.in
--- etc/jma-receipt.env.in	11 Nov 2002 13:26:40 -0000	1.8
+++ etc/jma-receipt.env.in	11 Nov 2002 13:32:31 -0000
@@ -28,6 +28,6 @@
 SITESCREENDIR=${SITEDIR}/screen
 SITESCRIPTSDIR=${SITEDIR}/scripts
 
-DBSTUB=/usr/local/panda/bin/dbstub
+DBSTUB=/usr/lib/panda/sbin/dbstub
 
 DBNAME=orca
Index: screen/H99.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/H99.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 H99.glade
--- screen/H99.glade	4 Sep 2002 07:03:03 -0000	1.1.1.1
+++ screen/H99.glade	11 Nov 2002 13:32:31 -0000
@@ -407,7 +407,7 @@
       <y>100</y>
       <width>60</width>
       <height>18</height>
-      <label>診療科　</label>
+      <label>診療科  </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/J04.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/J04.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 J04.glade
--- screen/J04.glade	4 Sep 2002 07:03:01 -0000	1.1.1.1
+++ screen/J04.glade	11 Nov 2002 13:32:31 -0000
@@ -2045,7 +2045,7 @@
       <y>80</y>
       <width>405</width>
       <height>20</height>
-      <label>名　　　　　　　　称</label>
+      <label>名                                称</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/K02.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/K02.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 K02.glade
--- screen/K02.glade	4 Sep 2002 07:03:01 -0000	1.1.1.1
+++ screen/K02.glade	11 Nov 2002 13:32:31 -0000
@@ -2546,7 +2546,7 @@
       <y>88</y>
       <width>340</width>
       <height>20</height>
-      <label>名　　称</label>
+      <label>名        称</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/K021.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/K021.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 K021.glade
--- screen/K021.glade	4 Sep 2002 07:03:01 -0000	1.1.1.1
+++ screen/K021.glade	11 Nov 2002 13:32:31 -0000
@@ -214,7 +214,7 @@
       <y>344</y>
       <width>56</width>
       <height>20</height>
-      <label>症　状</label>
+      <label>症    状</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/K023.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/K023.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 K023.glade
--- screen/K023.glade	4 Sep 2002 07:02:59 -0000	1.1.1.1
+++ screen/K023.glade	11 Nov 2002 13:32:31 -0000
@@ -1939,7 +1939,7 @@
       <y>36</y>
       <width>268</width>
       <height>20</height>
-      <label>名　　称</label>
+      <label>名        称</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/K05.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/K05.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 K05.glade
--- screen/K05.glade	4 Sep 2002 07:03:02 -0000	1.1.1.1
+++ screen/K05.glade	11 Nov 2002 13:32:31 -0000
@@ -2444,7 +2444,7 @@
       <y>80</y>
       <width>405</width>
       <height>20</height>
-      <label>名　　　　　　　　称</label>
+      <label>名                                称</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/N01.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/N01.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 N01.glade
--- screen/N01.glade	4 Sep 2002 07:03:06 -0000	1.1.1.1
+++ screen/N01.glade	11 Nov 2002 13:32:31 -0000
@@ -604,7 +604,7 @@
       <y>40</y>
       <width>80</width>
       <height>18</height>
-      <label>保険者番号　</label>
+      <label>保険者番号  </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -636,7 +636,7 @@
       <y>120</y>
       <width>140</width>
       <height>16</height>
-      <label>保険者名称　　　　</label>
+      <label>保険者名称        </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -652,7 +652,7 @@
       <y>570</y>
       <width>140</width>
       <height>18</height>
-      <label>記号　　　　　　　　</label>
+      <label>記号                </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -668,7 +668,7 @@
       <y>520</y>
       <width>140</width>
       <height>18</height>
-      <label>電話番号　　　　　　</label>
+      <label>電話番号            </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -684,7 +684,7 @@
       <y>470</y>
       <width>140</width>
       <height>18</height>
-      <label>番地方書　　　　　　</label>
+      <label>番地方書            </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -700,7 +700,7 @@
       <y>420</y>
       <width>140</width>
       <height>18</height>
-      <label>住所　　　　　　　　</label>
+      <label>住所                </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -716,7 +716,7 @@
       <y>370</y>
       <width>140</width>
       <height>18</height>
-      <label>郵便番号　　　　　　</label>
+      <label>郵便番号            </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -732,7 +732,7 @@
       <y>320</y>
       <width>140</width>
       <height>18</height>
-      <label>保険番号　　　　　　</label>
+      <label>保険番号            </label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/P02.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/P02.glade,v
retrieving revision 1.7
diff -u -r1.7 P02.glade
--- screen/P02.glade	31 Oct 2002 07:35:26 -0000	1.7
+++ screen/P02.glade	11 Nov 2002 13:32:31 -0000
@@ -5364,7 +5364,7 @@
 	  <y>516</y>
 	  <width>44</width>
 	  <height>24</height>
-	  <label>通　常</label>
+	  <label>通    常</label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -6397,7 +6397,7 @@
 	  <name>label1768</name>
 	  <x>24</x>
 	  <y>24</y>
-	  <width>22</width>
+	  <width>26</width>
 	  <height>24</height>
 	  <label>FAX</label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
Index: screen/P02D.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/P02D.glade,v
retrieving revision 1.2
diff -u -r1.2 P02D.glade
--- screen/P02D.glade	25 Sep 2002 11:21:21 -0000	1.2
+++ screen/P02D.glade	11 Nov 2002 13:32:31 -0000
@@ -447,7 +447,7 @@
       <y>116</y>
       <width>120</width>
       <height>24</height>
-      <label>　　労働保険番号</label>
+      <label>        労働保険番号</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -563,7 +563,7 @@
       <y>424</y>
       <width>120</width>
       <height>24</height>
-      <label>　　　　損傷区分</label>
+      <label>            損傷区分</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -579,7 +579,7 @@
       <y>324</y>
       <width>120</width>
       <height>24</height>
-      <label>　事業場の所在地</label>
+      <label>   事業場の所在地</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -747,7 +747,7 @@
       <y>292</y>
       <width>120</width>
       <height>24</height>
-      <label>　　　事業の名称</label>
+      <label>      事業の名称</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -763,7 +763,7 @@
       <y>260</y>
       <width>120</width>
       <height>24</height>
-      <label>　　　傷病年月日</label>
+      <label>       傷病年月日</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -779,7 +779,7 @@
       <y>224</y>
       <width>120</width>
       <height>24</height>
-      <label>　　　　新継再別</label>
+      <label>              新継再別</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -795,7 +795,7 @@
       <y>224</y>
       <width>100</width>
       <height>24</height>
-      <label>　　　転帰事由</label>
+      <label>       転帰事由</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -811,7 +811,7 @@
       <y>260</y>
       <width>100</width>
       <height>24</height>
-      <label>　　療養開始日</label>
+      <label>    療養開始日</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -827,7 +827,7 @@
       <y>260</y>
       <width>120</width>
       <height>24</height>
-      <label>　　　療養終了日</label>
+      <label>          療養終了日</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -843,7 +843,7 @@
       <y>228</y>
       <width>120</width>
       <height>24</height>
-      <label>　　四肢特例区分</label>
+      <label>     四肢特例区分</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -939,7 +939,7 @@
       <y>152</y>
       <width>120</width>
       <height>24</height>
-      <label>　　年金証書番号</label>
+      <label>         年金証書番号</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -1067,7 +1067,7 @@
       <y>452</y>
       <width>120</width>
       <height>24</height>
-      <label>　　　療養開始日</label>
+      <label>        療養開始日</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -1186,7 +1186,7 @@
       <y>88</y>
       <width>212</width>
       <height>20</height>
-      <label>＜短期給付　傷病年金  自賠責＞</label>
+      <label>＜短期給付    傷病年金  自賠責＞</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/P02E.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/P02E.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 P02E.glade
--- screen/P02E.glade	4 Sep 2002 07:03:10 -0000	1.1.1.1
+++ screen/P02E.glade	11 Nov 2002 13:32:32 -0000
@@ -359,7 +359,7 @@
       <y>120</y>
       <width>52</width>
       <height>24</height>
-      <label>住　所</label>
+      <label>住    所</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -391,7 +391,7 @@
       <y>64</y>
       <width>52</width>
       <height>24</height>
-      <label>氏　名</label>
+      <label>氏    名</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
Index: screen/R98.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/R98.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 R98.glade
--- screen/R98.glade	4 Sep 2002 07:03:10 -0000	1.1.1.1
+++ screen/R98.glade	11 Nov 2002 13:32:32 -0000
@@ -2449,7 +2449,7 @@
       <y>50</y>
       <width>100</width>
       <height>20</height>
-      <label>氏　　名</label>
+      <label>氏        名</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/V02.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/V02.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 V02.glade
--- screen/V02.glade	4 Sep 2002 07:03:07 -0000	1.1.1.1
+++ screen/V02.glade	11 Nov 2002 13:32:32 -0000
@@ -3718,7 +3718,7 @@
       <height>16</height>
       <label>〜</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -3790,7 +3790,7 @@
       <height>18</height>
       <label>有効期間</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -3806,7 +3806,7 @@
       <height>18</height>
       <label>支払区分</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -3906,7 +3906,7 @@
       <height>18</height>
       <label>保険番号</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4167,7 +4167,7 @@
       <height>18</height>
       <label>受給者検証番号チェック区分</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4183,7 +4183,7 @@
       <height>18</height>
       <label>検証番号チェック区分</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4199,7 +4199,7 @@
       <height>18</height>
       <label>制度名</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4215,7 +4215,7 @@
       <height>18</height>
       <label>短縮制度名</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4231,7 +4231,7 @@
       <height>18</height>
       <label>　　　年齢終了</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4247,7 +4247,7 @@
       <height>18</height>
       <label>レセプト請求</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4303,7 +4303,7 @@
       <height>18</height>
       <label>点数単価</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4319,7 +4319,7 @@
       <height>18</height>
       <label>条件−年齢開始</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4335,7 +4335,7 @@
       <height>18</height>
       <label>法別番号</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4351,7 +4351,7 @@
       <height>18</height>
       <label>保険公費種別区分</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4367,7 +4367,7 @@
       <height>18</height>
       <label>公費主保区分</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4383,7 +4383,7 @@
       <height>18</height>
       <label>限定保険番号１</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4399,7 +4399,7 @@
       <height>18</height>
       <label>　　　　　　２</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -4415,7 +4415,7 @@
       <height>18</height>
       <label>　　　　　　３</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
Index: screen/W03.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/W03.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 W03.glade
--- screen/W03.glade	4 Sep 2002 07:03:06 -0000	1.1.1.1
+++ screen/W03.glade	11 Nov 2002 13:32:32 -0000
@@ -1729,7 +1729,7 @@
       <y>10</y>
       <width>100</width>
       <height>18</height>
-      <label>有効期間　　　</label>
+      <label>有効期間      </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/W04.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/W04.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 W04.glade
--- screen/W04.glade	4 Sep 2002 07:03:06 -0000	1.1.1.1
+++ screen/W04.glade	11 Nov 2002 13:32:32 -0000
@@ -397,7 +397,7 @@
       <y>10</y>
       <width>100</width>
       <height>18</height>
-      <label>有効期間　　　</label>
+      <label>有効期間      </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -493,7 +493,7 @@
 	  <y>55</y>
 	  <width>300</width>
 	  <height>18</height>
-	  <label>（０，１：英数字型　２：数字型）</label>
+	  <label>（０，１：英数字型    ２：数字型）</label>
 	  <justify>GTK_JUSTIFY_CENTER</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -682,7 +682,7 @@
 	  <y>120</y>
 	  <width>200</width>
 	  <height>18</height>
-	  <label>（１：和暦　２：西暦）　　　　</label>
+	  <label>（１：和暦    ２：西暦）        </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -698,7 +698,7 @@
 	  <y>176</y>
 	  <width>200</width>
 	  <height>18</height>
-	  <label>（１：する　２：しない）　　　</label>
+	  <label>（１：する    ２：しない）      </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -746,7 +746,7 @@
 	  <y>120</y>
 	  <width>100</width>
 	  <height>18</height>
-	  <label>年区分　　　　</label>
+	  <label>年区分        </label>
 	  <justify>GTK_JUSTIFY_CENTER</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -762,7 +762,7 @@
 	  <y>24</y>
 	  <width>100</width>
 	  <height>18</height>
-	  <label>区分　　　　　</label>
+	  <label>区分          </label>
 	  <justify>GTK_JUSTIFY_CENTER</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -887,7 +887,7 @@
 	  <y>24</y>
 	  <width>100</width>
 	  <height>18</height>
-	  <label>区分　　　　　</label>
+	  <label>区分          </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -903,7 +903,7 @@
 	  <y>104</y>
 	  <width>100</width>
 	  <height>18</height>
-	  <label>前桁数　　　　</label>
+	  <label>前桁数        </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -919,7 +919,7 @@
 	  <y>152</y>
 	  <width>100</width>
 	  <height>18</height>
-	  <label>連番号桁数　　</label>
+	  <label>連番号桁数    </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
@@ -935,7 +935,7 @@
 	  <y>200</y>
 	  <width>100</width>
 	  <height>18</height>
-	  <label>後桁数　　　　</label>
+	  <label>後桁数        </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>False</wrap>
 	  <xalign>0.5</xalign>
Index: screen/W06.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/W06.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 W06.glade
--- screen/W06.glade	4 Sep 2002 07:03:07 -0000	1.1.1.1
+++ screen/W06.glade	11 Nov 2002 13:32:32 -0000
@@ -111,7 +111,7 @@
       <y>156</y>
       <width>100</width>
       <height>18</height>
-      <label>氏　　名</label>
+      <label>氏        名</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
@@ -145,7 +145,7 @@
       <height>18</height>
       <label>（社保）</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -161,7 +161,7 @@
       <height>24</height>
       <label>（国保）</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -209,7 +209,7 @@
       <height>18</height>
       <label>専門科コード１</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -225,7 +225,7 @@
       <height>18</height>
       <label>　　　　　　２</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -241,7 +241,7 @@
       <height>18</height>
       <label>　　　　　　３</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -257,7 +257,7 @@
       <height>18</height>
       <label>　　　　　　４</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -273,7 +273,7 @@
       <height>18</height>
       <label>　　　　　　５</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -289,7 +289,7 @@
       <height>16</height>
       <label>業務処理権限</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -305,7 +305,7 @@
       <height>18</height>
       <label>（M00）</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -321,7 +321,7 @@
       <height>18</height>
       <label>（M01）</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -337,7 +337,7 @@
       <height>18</height>
       <label>（M02）</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -353,7 +353,7 @@
       <height>18</height>
       <label>管理者権限</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -363,13 +363,13 @@
     <widget>
       <class>GtkLabel</class>
       <name>label226</name>
-      <x>56</x>
+      <x>75</x>
       <y>676</y>
       <width>100</width>
       <height>16</height>
       <label>（マスタ更新）</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
Index: screen/W07.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/W07.glade,v
retrieving revision 1.2
diff -u -r1.2 W07.glade
--- screen/W07.glade	10 Oct 2002 02:46:39 -0000	1.2
+++ screen/W07.glade	11 Nov 2002 13:32:32 -0000
@@ -351,9 +351,9 @@
 	  <y>128</y>
 	  <width>300</width>
 	  <height>16</height>
-	  <label>２：薬剤情報提供料（一般）　　　　　　　</label>
+	  <label>２：薬剤情報提供料（一般）              </label>
 	  <justify>GTK_JUSTIFY_CENTER</justify>
-	  <wrap>False</wrap>
+	  <wrap>True</wrap>
 	  <xalign>0.5</xalign>
 	  <yalign>0.5</yalign>
 	  <xpad>0</xpad>
@@ -367,9 +367,9 @@
 	  <y>160</y>
 	  <width>300</width>
 	  <height>16</height>
-	  <label>３：薬剤情報提供料（老人　手帳記載あり）</label>
+	  <label>３：薬剤情報提供料（老人    手帳記載あり）</label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
-	  <wrap>False</wrap>
+	  <wrap>True</wrap>
 	  <xalign>0.5</xalign>
 	  <yalign>0.5</yalign>
 	  <xpad>0</xpad>
@@ -383,9 +383,9 @@
 	  <y>192</y>
 	  <width>300</width>
 	  <height>16</height>
-	  <label>４：薬剤情報提供料（老人　手帳記載なし）</label>
+	  <label>４：薬剤情報提供料（老人    手帳記載なし）</label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
-	  <wrap>False</wrap>
+	  <wrap>True</wrap>
 	  <xalign>0.5</xalign>
 	  <yalign>0.5</yalign>
 	  <xpad>0</xpad>
@@ -559,9 +559,9 @@
 	  <y>96</y>
 	  <width>300</width>
 	  <height>16</height>
-	  <label>１：調剤技術基本料　　　　　　　　　　　</label>
+	  <label>１：調剤技術基本料                      </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
-	  <wrap>False</wrap>
+	  <wrap>True</wrap>
 	  <xalign>0.5</xalign>
 	  <yalign>0.5</yalign>
 	  <xpad>0</xpad>
Index: screen/W10.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/W10.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 W10.glade
--- screen/W10.glade	4 Sep 2002 07:02:59 -0000	1.1.1.1
+++ screen/W10.glade	11 Nov 2002 13:32:32 -0000
@@ -297,7 +297,7 @@
       <height>16</height>
       <label>〜</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -341,7 +341,7 @@
       <height>18</height>
       <label>有効期間</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
Index: screen/W11.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/W11.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 W11.glade
--- screen/W11.glade	4 Sep 2002 07:03:08 -0000	1.1.1.1
+++ screen/W11.glade	11 Nov 2002 13:32:32 -0000
@@ -1482,7 +1482,7 @@
       <y>10</y>
       <width>100</width>
       <height>18</height>
-      <label>有効期間　　　</label>
+      <label>有効期間      </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/W12.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/W12.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 W12.glade
--- screen/W12.glade	4 Sep 2002 07:03:08 -0000	1.1.1.1
+++ screen/W12.glade	11 Nov 2002 13:32:32 -0000
@@ -411,7 +411,7 @@
       <y>100</y>
       <width>150</width>
       <height>16</height>
-      <label>傷病名編集区分　　</label>
+      <label>傷病名編集区分    </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -467,7 +467,7 @@
       <y>200</y>
       <width>150</width>
       <height>16</height>
-      <label>転帰日編集区分　　</label>
+      <label>転帰日編集区分    </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -933,10 +933,10 @@
       <name>label223</name>
       <x>112</x>
       <y>444</y>
-      <width>136</width>
+      <width>146</width>
       <height>32</height>
-      <label>県別レセプト編集　　
-　　　プログラム名</label>
+      <label>県別レセプト編集        
+            プログラム名</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/X01.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/X01.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 X01.glade
--- screen/X01.glade	4 Sep 2002 07:03:06 -0000	1.1.1.1
+++ screen/X01.glade	11 Nov 2002 13:32:32 -0000
@@ -461,7 +461,7 @@
       <y>90</y>
       <width>100</width>
       <height>18</height>
-      <label>チェック区分　</label>
+      <label>チェック区分  </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -477,7 +477,7 @@
       <y>130</y>
       <width>100</width>
       <height>18</height>
-      <label>診療コード　　</label>
+      <label>診療コード    </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/X02.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/X02.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 X02.glade
--- screen/X02.glade	4 Sep 2002 07:03:06 -0000	1.1.1.1
+++ screen/X02.glade	11 Nov 2002 13:32:32 -0000
@@ -1467,7 +1467,7 @@
       <y>56</y>
       <width>100</width>
       <height>18</height>
-      <label>有効期間　　　</label>
+      <label>有効期間      </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -1547,7 +1547,7 @@
       <y>16</y>
       <width>100</width>
       <height>18</height>
-      <label>チェック区分　</label>
+      <label>チェック区分  </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
@@ -1563,7 +1563,7 @@
       <y>130</y>
       <width>445</width>
       <height>20</height>
-      <label>名　　　　　　　　称</label>
+      <label>名                                称</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/Z02.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/Z02.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Z02.glade
--- screen/Z02.glade	4 Sep 2002 07:03:05 -0000	1.1.1.1
+++ screen/Z02.glade	11 Nov 2002 13:32:32 -0000
@@ -989,7 +989,7 @@
       <height>18</height>
       <label>レセプト編集情報</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -1005,7 +1005,7 @@
       <height>18</height>
       <label>カラム位置</label>
       <justify>GTK_JUSTIFY_RIGHT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -1062,7 +1062,7 @@
       <height>18</height>
       <label>有効年月日</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -1087,7 +1087,7 @@
       <height>18</height>
       <label>漢字名称</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -1103,7 +1103,7 @@
       <height>18</height>
       <label>カナ名称</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
@@ -1119,7 +1119,7 @@
       <height>18</height>
       <label>正式名称</label>
       <justify>GTK_JUSTIFY_LEFT</justify>
-      <wrap>True</wrap>
+      <wrap>False</wrap>
       <xalign>0.5</xalign>
       <yalign>0.5</yalign>
       <xpad>0</xpad>
Index: screen/Z03.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/Z03.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Z03.glade
--- screen/Z03.glade	4 Sep 2002 07:03:05 -0000	1.1.1.1
+++ screen/Z03.glade	11 Nov 2002 13:32:32 -0000
@@ -567,7 +567,7 @@
 	  <y>536</y>
 	  <width>64</width>
 	  <height>24</height>
-	  <label>〜　上限</label>
+	  <label>〜    上限</label>
 	  <justify>GTK_JUSTIFY_RIGHT</justify>
 	  <wrap>True</wrap>
 	  <xalign>0.5</xalign>
@@ -5023,7 +5023,7 @@
 	  <y>112</y>
 	  <width>188</width>
 	  <height>22</height>
-	  <label>点数欄集計先識別　入院外　</label>
+	  <label>点数欄集計先識別    入院外  </label>
 	  <justify>GTK_JUSTIFY_LEFT</justify>
 	  <wrap>True</wrap>
 	  <xalign>0.5</xalign>
Index: screen/Z05.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/Z05.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Z05.glade
--- screen/Z05.glade	4 Sep 2002 07:03:05 -0000	1.1.1.1
+++ screen/Z05.glade	11 Nov 2002 13:32:32 -0000
@@ -1235,7 +1235,7 @@
       <y>364</y>
       <width>96</width>
       <height>16</height>
-      <label>〜　上限年齢</label>
+      <label>〜    上限年齢</label>
       <justify>GTK_JUSTIFY_RIGHT</justify>
       <wrap>True</wrap>
       <xalign>0.5</xalign>
Index: screen/Z06.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/Z06.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Z06.glade
--- screen/Z06.glade	4 Sep 2002 07:03:10 -0000	1.1.1.1
+++ screen/Z06.glade	11 Nov 2002 13:32:32 -0000
@@ -601,7 +601,7 @@
       <y>200</y>
       <width>80</width>
       <height>24</height>
-      <label>服用時点　</label>
+      <label>服用時点  </label>
       <justify>GTK_JUSTIFY_LEFT</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/Z90.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/Z90.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Z90.glade
--- screen/Z90.glade	4 Sep 2002 07:03:08 -0000	1.1.1.1
+++ screen/Z90.glade	11 Nov 2002 13:32:32 -0000
@@ -459,7 +459,7 @@
       <y>72</y>
       <width>344</width>
       <height>24</height>
-      <label>名　　　　　　　　称</label>
+      <label>名                                称</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
Index: screen/Z91.glade
===================================================================
RCS file: /var/lib/cvs/jma-receipt/screen/Z91.glade,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Z91.glade
--- screen/Z91.glade	4 Sep 2002 07:03:08 -0000	1.1.1.1
+++ screen/Z91.glade	11 Nov 2002 13:32:32 -0000
@@ -303,7 +303,7 @@
       <y>96</y>
       <width>64</width>
       <height>24</height>
-      <label>〜　終了</label>
+      <label>〜    終了</label>
       <justify>GTK_JUSTIFY_CENTER</justify>
       <wrap>False</wrap>
       <xalign>0.5</xalign>
